draft.md

```diff
@@ -116,10 +116,10 @@ API publishers SHOULD use following values for the field:
   - "warn": healthy, with some concerns.
 
   The value of the status field is case-insensitive and is tightly related with
-  the HTTP response code returned by the health endpoint. For “pass” and “warn”
-  statuses, HTTP response code in the 2xx-3xx range MUST be used. For “fail”
-  status, HTTP response code in the 4xx-5xx range MUST be used. In case of the
-  “warn” status, endpoints SHOULD return HTTP status in the 2xx-3xx range, and
+  the HTTP response code returned by the health endpoint. For “pass” status,
+  HTTP response code in the 2xx-3xx range MUST be used. For “fail” status,
```

Comment:
I was thinking of giving here at least one specific [suggested] example code for each status case. Wdyt?
pass: 200
warn: [207](https://httpstatuses.com/207)?
fail: [424](https://httpstatuses.com/424) maybe?

Reply:
207 is WebDAV. Not all web frameworks (clients or servers) will have an understanding of the WebDAV status codes.

This RFC should probably be conservative with the other HTTP-oriented RFCs it relies on.

Reply:
Re: “warn” ... some people will put thresholds in their health checks, but, as a best practice, I personally will avoid putting the service metrics thresholds in my code, and I will put the thresholds in my external monitoring and control systems.

To me “warn” should be oriented towards “degraded” states. Think failing health checks that are neither liveness checks nor readiness checks. When liveness or readiness health checks are failing, I am “down.” When non-liveness and non-readiness health checks are failing, I am “warn”, which to me means “up and able to serve but degraded (relying on sane fallbacks or recording things for deferred batch operations upon service restoration, etc.).”